www/docs/mp/index.php

```diff
@@ -671,13 +671,32 @@
     }
 
 } catch (MySociety\TheyWorkForYou\MemberException $e){
+    global $this_page;
+
+    switch($this_page) {
+    case 'mla':
+        $rep = 'MLA';
+        $SEARCHURL = '/postcode/';
+        $MPSURL = new \URL('mlas');
+        break;
+    case 'msp':
+        $rep = 'MSP';
+        $SEARCHURL = '/postcode/';
+        $MPSURL = new \URL('msps');
+        break;
+    default:
+        $rep = 'MP';
+        $SEARCHURL = new \URL('mp');
+        $SEARCHURL = $SEARCHURL->generate();
```

Comment:
I think this could use `/postcode/` in all cases? Might as well stick with the single redirector unless going straight to MP does something different.